File: unidad5/Unidad5programa5.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.unidad5programa5;
import java.util.Scanner;
/**
 *
 * @author Emiliano
 */
public class Unidad5programa5 {

    public static void main(String[] args) {
       Scanner sc = new Scanner(System.in);
        System.out.print("escribe una palabra: ");
        String palabra = sc.nextLine();
        System.out.println(palabra+" esta palabra esta conformada por: "+palabra(palabra)+" letras");
    }
    public static int palabra(String x){
        int p = x.length();
        return p;
    }
}
    

File: unidad5/unidad5programa5.cs
using System;

namespace unidad5programa5 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {

           string p = "";
            Console.Write("escribe una palabra : ");
            p = Console.ReadLine();
            Console.WriteLine(p + " esta palabra esta conformada por : " + palabras(p) + " letras");
        }
        public static int palabras(string x)
        {
            int s = x.Length;
            return s;
        }
    }
}
    

